src/components/NavMenu.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react'
+
+import '../styles/home_nav.css'
+
+class NavMenu extends Component {
+  getDefaultProps() {
+    return {
+      isOpen: false
+    }
+  }
+
+  render() {
+    if(this.props.isOpen) {
```

Comment:
Please add spaces between opening and closing parentheses, and arguments, in a function invocation or control flow expression:
```js
fn(param1, param2) { /* ... */ }
if(expression) { /* ... */ }
```
should be
```js
fn( params1, param2 ) { /* ... */ }
if( expression ) { /* ... */ }
```